WordPress/Classes/ViewRelated/Post/WPStyleGuide+Posts.swift

```diff
@@ -0,0 +1,140 @@
+        view.backgroundColor = postCardBorderColor()
+    }
+
+    class func applyActionBarButtonStyle(_ button: UIButton) {
+        button.flipInsetsForRightToLeftLayoutDirection()
+        button.setImage(button.imageView?.image?.imageWithTintColor(grey()), for: .normal)
+        button.setTitleColor(grey(), for: .normal)
+        button.setTitleColor(darkGrey(), for: .highlighted)
+        button.setTitleColor(darkGrey(), for: .selected)
+    }
+
+    class func applySelectedBackgroundView(_ selectedBackgroundView: UIView, topMargin: CGFloat) {
+        let marginMask = UIView()
+        selectedBackgroundView.addSubview(marginMask)
+        marginMask.translatesAutoresizingMaskIntoConstraints = false
+        marginMask.leadingAnchor.constraint(equalTo: selectedBackgroundView.leadingAnchor).isActive = true
+        marginMask.topAnchor.constraint(equalTo: selectedBackgroundView.topAnchor).isActive = true
+        marginMask.trailingAnchor.constraint(equalTo: selectedBackgroundView.trailingAnchor).isActive = true
+        marginMask.heightAnchor.constraint(equalToConstant: topMargin).isActive = true
+        marginMask.backgroundColor = greyLighten30()
+    }
+
+    // MARK: - Attributed String Attributes
+
+    class var postCardTitleAttributes: [NSAttributedString.Key: Any] {
+        let font = notoBoldFontForTextStyle(.headline)
+        let lineHeight = 1.4 * font.pointSize
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.minimumLineHeight = lineHeight
+        paragraphStyle.maximumLineHeight = lineHeight
+        return [.paragraphStyle: paragraphStyle, .font: font]
+    }
+
+    class var postCardSnippetAttributes: [NSAttributedString.Key: Any] {
+        let textStyle: UIFont.TextStyle = UIDevice.isPad() ? .callout : .subheadline
+        let font = notoFontForTextStyle(textStyle)
+        let lineHeight = 1.6 * font.pointSize
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.minimumLineHeight = lineHeight
+        paragraphStyle.maximumLineHeight = lineHeight
+        return [.paragraphStyle: paragraphStyle, .font: font]
+    }
+
+    // MARK: - Nav Bar Styles
+
+    class var navigationBarButtonRect: CGRect {
+        return CGRect(x: 0, y: 0, width: 30, height: 30)
+    }
+
+    class var spacingBetweeenNavbarButtons: CGFloat {
+        return 40
+    }
+
+    class func buttonForBar(with image: UIImage,
+                            target: Any?, selector: Selector) -> WPButtonForNavigationBar {
+        let button = WPButtonForNavigationBar(frame: navigationBarButtonRect)
+        button.tintColor = .white
+        button.setImage(image, for: .normal)
+        button.addTarget(target, action: selector, for: .touchUpInside)
+        button.removeDefaultLeftSpacing = true
+        button.removeDefaultRightSpacing = true
+        button.rightSpacing = spacingBetweeenNavbarButtons / 2
```

Comment:
I realise this value matches what was in ObjC, but what do you think about rounding the result of `spacingBetweeenNavbarButtons / 2`?

Reply:
Any special reason to round this value?

I mean, `spacingBetweeenNavbarButtons` is now `40`, which means that this value will always be `20` regardless of rounding it or not.

But if the value changes for, let's say, `41`, we want a rounded number to be used as spacing?

Reply:
I think with `40`, it's fine, just thinking of the weird case if it becomes `39` and `19.5` isn't pixel-aligned on `@3x` devices. Anyway—pretty minor. I wouldn't hold up this PR for this particular comment ✅